src/app/core/trainee/trainee.service.ts

```diff
@@ -31,7 +31,7 @@ export class TraineeService {
       .set("sortColumn", snapshot.sort.active)
       .set("sortOrder", snapshot.sort.direction)
       .set("pageNumber", snapshot.pageIndex.toString())
-      .set("underNotice", snapshot.underNotice.toString());
+      .set("underNotice", snapshot.filter === "underNotice" ? "true" : "false");
```

Comment:
Can't we set boolean flags, doesn't seems good to use boolean flags as strings?

Reply:
i agree it would be cleaner @babul3012 but unfortunately the `.set()` method on angular's `HttpParams` class requires the value to be a string